adage/adage/config.py.template

```diff
@@ -12,35 +12,51 @@ BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 REPO_ROOT = os.path.dirname(BASE_DIR)
 
 # AWS credentials and configuration
-AWS_CONFIG = {
+AWS_DEPLOY = {
     'aws_access_key_id':        '<config-secret>',
     'aws_secret_access_key':    '<config-secret>',
     'region_name':              'us-east-1',
-    'rds_conn':      {
-                         'NAME': 'adage_db',
-                         'USER': 'adage_db',
-                         'PASSWORD': '<config-secret>',
-                         'HOST': '<configure>',
-                         'PORT': '5432',
-                     },
-    'ec2_params':    {
-                         # 'DryRun': True,
-                         'ImageId': 'ami-d05e75b8',
-                         'MinCount': 1, 'MaxCount': 1,
-                         'KeyName': '<configure>',
-                         'SecurityGroups':  ['<configure>'],
-                         'InstanceType': 't2.micro',
-                     },
-    'host_conn':     {
-                         'host': '<configure>',  # <-- our elastic IP address
-                         'user': 'ubuntu',
-                         'keyfile': '<configure>',
-                     }
+    # 'dbmaster' holds credentials for the RDS instance used on AWS
+    'dbmaster': {
+        # database user with CREATE DATABASE & ROLE privs
+        # these are used to bootstrap a new database as
+        # specified in CONFIG['databases']['default']
+        'NAME': 'adage_db',
+        'USER': 'adage_db',
+        'PASSWORD': '<config-secret>',
+        'HOST': '<configure>',
+        'PORT': '5432',
+    },
+    'ec2_params': {
+        # 'DryRun': True,
```

Comment:
Oh, I can take that out. It's left over from testing. A "dry run" will walk through the steps of launching an EC2 instance, but won't actually spin it up. Main difference: a dry run is free, spinning up an instance costs at least a few pennies. We're well past the point of figuring out how to launch instances correctly, though, so it should come out.

- [ ] remove 'DryRun' parameter from config.py.template's `ec2_params`